.eslintrc.js

```diff
@@ -32,7 +32,7 @@ module.exports = {
         'ts-ignore': 'allow-with-description',
       },
     ],
-    'arrow-body-style': ['error', 'always'],
+    curly: ['error', 'all'],
```

Comment:
This was done to match AirBNB standard, and to allow for shorter code in many callbacks.